include/abb_librws/rws_poco_client.h

```diff
@@ -306,14 +306,15 @@ class POCOClient
   POCOResult httpDelete(const std::string uri);
   
   /**
-   * \brief A method for resetting the timeout to the default value.
-   */
-  void resetTimeout() { http_client_session_.setTimeout(Poco::Timespan(DEFAULT_TIMEOUT)); }
-  
-  /**
-   * \brief A method for setting the timeout to a long value.
+   * \brief A method for setting the HTTP communication timeout.
+   *
+   * \param timeout for the HTTP communication timeout [microseconds].
    */
-  void setLongTimeout() { http_client_session_.setTimeout(Poco::Timespan(LONG_TIMEOUT)); }
+  void setHTTPTimeout(const Poco::Int64 timeout)
+  {
+    http_client_session_.setTimeout(Poco::Timespan(timeout));                              
+    http_client_session_.reset();
```

Comment:
What is the `reset()` for?

Reply:
The `HTTPClientSession` instance dosen't seem to apply the new timeout without the `reset()`.

Reply:
Would it make sense to add a note about that to the documentation on `setHTTPTimeout(..)`?

Reply:
I will add a note about it for documentation's sake.

A user will probably never notice anything, because the thing that happens when the client is reset is that the RWS server (in the robot controller) sends out an updated cookie number but the RWS session id is not changed. All of this is taken care of in the background.